docs/document/docs/components/desktop/tabs.md

```diff
@@ -0,0 +1,124 @@
+---
+id: tabs
+title: Tabs
+sidebar_label: Tabs
+---
+
+## Overview
+
+A multiple tabs component that can switch displaying contents with buttons.
+
+<div class="sample-container" id="tabs">
+  <div id="sample-container__components"></div>
+</div>
+<script src="/js/samples/desktop/tabs.js"></script>
+
+---
+
+## Specification
+
+### Property
+Here is a list of properties that can be used for modifying the component:
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| id | string | ""  | Component id name | |
+| value | string | ""  | Component id name | The first tab will be displayed if the `value` is unspecified or there is no matching value in items. |
+| borderVisible | boolean | true  | Show/Hide the border surrounding the content | |
+| visible | boolean | true | Show/Hide the component | |
+| items | Array\<Item> | [] | List of items for tab will be displayed | If the items is not an array, the error message 'items' property is not array. will throw in window console. |
+| Item.content | string/HTMLElement | "" | Tab content | |
+| Item.label | string | "" | Tab name | |
+| Item.value | string | "" | Tab key | `Item.value` is unique and required.<br>Will result an error if the `Item.value` is duplicated in `items` or not specified |
+| Item.disabled | boolean | false | Enable/Disable the tab | |
+| Item.visible | boolean | true | Show/Hide the tab | |
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | Event handler of when selected tab is changed |  It will pass the event object as the argument<br><br>You can receive the following values in event.detail<br><li>event.detail.oldValue : “value” before the change</li><li>event.detail.value : “value” after the change</li> |
```

Comment:
```suggestion
| change | function | Event handler of when selected tab is changed |  It will pass the event object as the argument<br><br>You can receive the following values in event.detail<br><li>event.detail.oldValue : "value" before the change</li><li>event.detail.value : "value" after the change</li> |
```